src/lib/output/themes/MarkedPlugin.ts

```diff
@@ -195,6 +195,19 @@ output file :
         if (!markedOptions.renderer) {
             markedOptions.renderer = new Marked.Renderer();
 
+            markedOptions.renderer.link = (href, title, text) => {
+                // Prefix the #anchor links `#md:`.
+                href =
+                    href
+                        ?.replace(/^#(?:md:)?(.+)/, "#md:$1")
+                        .replace(/"/g, "&quot;") || "";
+                let html = `<a href="${href}"`;
+                if (title != null)
+                    html += ` title="${title.replace(/"/g, "&quot;")}"`;
+                html += `>${text}</a>`;
```

Comment:
Awkward to have to construct the HTML tag manually - not sure if marked or typedoc provides an API for this?

Reply:
marked doesn't, typedoc does via the jsx templating engine, I'll make that change after merging

Reply:
Actually... I think we can revert this annoying prefixing now! The minimal theme was removed a while ago, so I'm not sure there's a reason for it anymore...

Reply:
... sadly not. I'd like to rework the renderer at some point to make this possible, but headings in comments still get linked. It feels like there ought to be some other non-terrible way to do this, that doesn't make us write different headings than npm/github does when rendering...

Reply:
Perhaps going the other direction would work; adding prefixes to the headings which typedoc "owns" (generates) and leaving the regular markdown ids unprefixed?

Reply:
I considered that when first adding the prefix, but typedoc generates a *lot* of anchors, there are generally far, far fewer markdown header links.